process.php

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];

```

Comment:
this is that post array at work. your solution won't work because we don't have that other file.